docs/getting_started/about_composer.md

```diff
@@ -43,26 +43,37 @@ By doing it this way you will need to execute further Composer commands using `p
     -   as Username – your Installation key found above on the *"Maintenance and Support agreement details"* page in the service portal
     -   as Password – the token password you retrieved in step 3.
 
-    #### Support agreement expiry
+    #### Note: Support agreement expiry
 
-    If your Support agreement expires, your authentication token(s) will no longer work. They will become active again if the agreement is renewed, but this process may take up to 24 hours. (If the agreement is renewed before the expiry date, there will be no disruption of service.)
+    If your Support agreement expires, your authentication token(s) will no longer work. They will become active again if the agreement is renewed, but this process may take up to 24 hours. _(If the agreement is renewed before the expiry date, there will be no disruption of service.)_
 
-    ### Optional: Save authentication information in auth.json to avoid repeatedly typing it
+    ### Tip: Save authentication token in auth.json to avoid repeatedly typing it
 
     Composer will ask to do this for you on updates, however if it is disabled, you can create an `auth.json` file manually in one of the following ways:
 
-    * Option A: Store your credentials in the project directory:
+    * Option A: Store your credentials in the project directory _(but for security reasons avoid checking it in to git)_:
 
     ``` bash
     composer config http-basic.updates.ez.no <installation-key> <token-password>
     ```
 
-    * Option B: If you'd rather want to install it globally in [COMPOSER\_HOME](https://getcomposer.org/doc/03-cli.md#composer-home) directory for machine-wide use:
+    * Option B: If you only have one project on the machine/server/vm, and you'd rather want to install it globally in [COMPOSER\_HOME](https://getcomposer.org/doc/03-cli.md#composer-home) directory for machine-wide use:
 
     ``` bash
     composer config --global http-basic.updates.ez.no <installation-key> <token-password>
     ```
 
+    ### Tip: Usage of authentication token with `composer create-project`
+
+    If you have several projects setup on your machine, they should all use different tokens set in auth.json file in project directory. 
+    
+    But in order to install a new project using `composer create-project` to get latest version of eZ Platform Enterprise, you'll need to tell composer which token to use before the project folder is created in the first place. This can be done in the following way:
+
+    ``` bash
+    COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer composer create-project --keep-vcs ezsystems/ezplatform-ee my-new-ee-project
```

Comment:
@andrerom _composer_ is duplicated in this command. 🙂

Reply:
Indeed, fixed in 855dde6 👍